console/console-init/ui/src/modules/device/components/CreateDeviceButton/CreateDeviceButton.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import { Dropdown, DropdownToggle, DropdownItem } from "@patternfly/react-core";
+import { CaretDownIcon } from "@patternfly/react-icons";
+
+export interface ICreateDeviceButtonProps {
+  inputInfo: () => void;
+  inputJSON: () => void;
+}
+
+export const CreateDeviceButton: React.FunctionComponent<ICreateDeviceButtonProps> = ({
+  inputInfo,
+  inputJSON
+}) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const dropdownItems = [
+    <DropdownItem
+      onClick={() => inputInfo()}
```

Comment:
Can we directly call inputInfo here, without the arrow function as there are no arguments.

Reply:
The existing components handle onClick of Dropdown items in a similar manner, that is the reason of following this convention.